microros_extensions/arm_toolchain.cmake.in

```diff
@@ -4,7 +4,7 @@ set(CMAKE_SYSTEM_NAME Generic)
 # set(CMAKE_SYSTEM_PROCESSOR arm)
 set(CMAKE_CROSSCOMPILING 1)
 set(CMAKE_TRY_COMPILE_TARGET_TYPE STATIC_LIBRARY)
-set(PLATFORM_NAME "nuttx")
+# set(PLATFORM_NAME "Zephyr")
```

Comment:
Why is it commented? I think the platform name should be `zephyr` in lowercase.

Reply:
Fixed, although in dashing branch we don't have zephyr handling for the Micro XRCE-Client: https://github.com/eProsima/Micro-XRCE-DDS-Client/blob/46ee5bcd3fa1a07cdbe41ecb2b9a9d8c4b2a874d/CMakeLists.txt#L117